pkgs/applications/misc/crumbs/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, automake, autoconf, gzip, ruby, bash }:
+
+stdenv.mkDerivation rec {
+  name = "crumbs";
```

Comment:
please use `pname` instead of `name`.

Reply:
my commit replacing `name` with `pname` seems to have failed the package listing tests. Is there some thing I'm missing?